### PR TITLE
Add Portworx demo videos to the docs site

### PR DESCRIPTION
**Pull request by andreipope**
Add Portworx demo videos to the docs site

### CODE REVIEW

**Comment by andreipope** [comment]
@nathan-px @wallnerryan Hey guys, please review this PR when you get a chance.
Note that this video is private https://www.youtube.com/watch?v=vCnUczaSXDA.

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 1]
```diff
@@ -260,3 +260,22 @@ mysqlmigrationschedule-weekly-2019-02-14-221351 1d
 ```
 
 Once the MigrationSchedule object is deleted, all the associated Migration objects should be deleted as well.
+
+
+## Useful References
+
+* Video tutorial - Set up multi-cluster application failover with CockroachDB:
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: Ordinarily, a list makes sense here. But given how much screen real-estate these videos take-up I think we shouldn't use a bulleted list. 

Let's subhead them with the video title instead.

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md [hunk 2]
```diff
@@ -260,3 +260,22 @@ mysqlmigrationschedule-weekly-2019-02-14-221351 1d
 ```
 
 Once the MigrationSchedule object is deleted, all the associated Migration objects should be deleted as well.
+
+
+## Useful References
```
[the remark this answers is not in the document]
Comment (andreipope) [reply content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: @nathan-px Yes, it's probably better.

File: content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md (hunk 1 above)
Comment (andreipope) [reply content/portworx-install-with-kubernetes/disaster-recovery/async-dr/_index.md]: @nathan-px Done.

**Comment by andreipope** [comment]
@nathan-px Hey, Nathan, this PR is updated. Please review when you get the chance. Thanks!

**Comment by andreipope** [comment]
@wallnerryan Just wanted to bump this to your attention

**Pull request merged**